extensions/amp-form/0.1/form-validators.js

```diff
@@ -286,8 +286,10 @@ export class AbstractCustomValidator extends FormValidator {
   onInteraction(event) {
     const input = dev().assertElement(event.target);
     const shouldValidate = this.shouldValidateOnInteraction(input);
+    const isInvalid = input.checkValidity && !input.checkValidity();
```

Comment:
good catch, anything with `tabIndex > 0` could trigger `blur` and does not mean it is an input.

I recommend moving the `input.checkValidity` check to the `const shouldValidate` constant. Essentially using  `input.checkValidity` to detect whether target was an input or not.

Reply:
That's a good call, maybe it would make sense in the body of `shouldValidateOnInteraction`, since it takes an input argument and it should not validate when interacting with an element without a `checkValidity` method.

Reply:
:+1:

Reply:
Ended up taking your recommendation!